.golangci.yaml

```diff
@@ -65,6 +65,9 @@ issues:
     - linters:
         - golint
       text: "a blank import should be only in a main or test package"
+    - linters:
+        - govet
+      text: "shadow: declaration of"
```

Comment:
Doesn't it work as follow?
https://github.com/golangci/golangci-lint/blob/58234f055cd8a4ba71ce6dafb95d7fa665bc8d39/.golangci.example.yml#L242-L243

Reply:
Ok, done,